health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -2667,67 +2576,89 @@
       }
     }
-    else{
+    else {
       throwError("COMMON", 400, "USER_CREDENTIAL_GENERATION_ERROR", "User credential generation failed");
     }
   }
 }
 
-async function createAllResources(campaignDetails: any,parentCampaign : any, useruuid: string) {
+async function createAllResources(campaignDetails: any, parentCampaign: any, useruuid: string) {
   const { maxAttemptsForResourceCreationOrMapping, waitTimeOfEachAttemptOfResourceCreationOrMappping } = config?.resourceCreationConfig
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)